electron/gog/library.ts

```diff
@@ -632,7 +632,11 @@ export async function runGogdlCommand(
   }
 
   return new Promise((res, rej) => {
-    const child = spawn(bin, commandParts, { cwd: dir, env: env })
+    const child = spawn(bin, commandParts, {
+      cwd: dir,
+      env: env,
+      shell: getShell()
```

Comment:
Are you sure this doesn't bring back the quote issues?

It might be better to really only run in a shell if we're on MacOS
So
```suggestion
      shell: isMac
```